api/server/collections.ts

```diff
@@ -0,0 +1,37 @@
+import { MongoObservable } from 'meteor-rxjs';
```

Comment:
Also the upcoming PR is heavily based on these paths so I'd rather change it in the next PRs, not this one, since it will screw up everything.